scalyr_agent/agent_main.py

```diff
@@ -1343,19 +1343,27 @@ def __verify_can_write_to_logs_and_data(self, config):
         @type config: Configuration
         """
         if not os.path.isdir(config.agent_log_path):
-            raise Exception(
-                "The agent log directory '%s' does not exist." % config.agent_log_path
-            )
+            if self.__controller.install_type != DEV_INSTALL:
```

Comment:
This change seems reasonable to me, but to be on the safe side, I would invert the ``if`` condition and call ``os.makedirs`` if it's a dev install.